hwy/ops/wasm_256-inl.h

```diff
@@ -1267,6 +1267,89 @@ HWY_API Vec256<T> ReverseBlocks(D /* tag */, const Vec256<T> v) {
   return SwapAdjacentBlocks(v);  // 2 blocks, so Swap = Reverse
 }
 
+// ------------------------------ Per4LaneBlockShuffle
+namespace detail {
+
+template <size_t kIdx0, size_t kIdx1, size_t kIdx2, size_t kIdx3, class V>
+HWY_INLINE V Per4LaneBlockShuffle(hwy::SizeTag<kIdx0> /*idx_0_tag*/,
+                                  hwy::SizeTag<kIdx1> /*idx_1_tag*/,
+                                  hwy::SizeTag<kIdx2> /*idx_2_tag*/,
+                                  hwy::SizeTag<kIdx3> /*idx_3_tag*/,
+                                  hwy::SizeTag<1> /*lane_size_tag*/,
```

Comment:
We could handle sizes <= 4 by calling ::Per4LaneBlockShuffle() for each 128-bit half?